### ISSUE TITLE
Spec not update on cocoapods.org

### ISSUE THREAD

**Issue by emitelma** [John-Lluch/SWRevealViewController#420]
still showing v2.3

**Comment by schayes04** [comment]
Yes I believe @John-Lluch as the owner of this library needs to push the update to the Cocoapods specs

**Comment by emitelma** [comment]
@John-Lluch still showing 2.3 in cocoapods.org. Could you please fix?

**Comment by ogubariev** [comment]
@John-Lluch Why you don't update your library on cocoapods.org?

**Comment by John-Lluch** [comment]
It doesn't work for me. When I attempt to perform the push it says something about not having enough permissions and do not really know how to solve it because I am not an user of cocoapods. I also attempted to re-install cocoapods to no avail. Maybe someone can give me a hint on why this could happen?

**Comment by jhersh** [comment]
The repo owner, @John-Lluch, will need to register with CocoaPods trunk and push a new version to the master specs repo using `pod trunk push`. See http://guides.cocoapods.org/making/getting-setup-with-trunk.html

**Comment by emitelma** [comment]
Hi, @John-Lluch any luck?

**Comment by linked67** [comment]
Still 2.3... please fix it

**Comment by pepejeria** [comment]
Hi @John-Lluch, is there anyway that we can help to push the latest version to Cocoapods?
